src/features/common.step.ts

```diff
@@ -175,27 +300,27 @@ Then('I expect that the element with role {string} and order {int} is visible',
   await expect(basePage.getPage().getByRole(text, { exact: true }).nth(order)).toBeVisible()
 })
 
-Then('I expect that I go back to the {string} page', async ({ basePage }, pageName) => {
-  // Check if whether we are on vaults page
-  await expect(
-    basePage.getPage().getByRole('heading', {
-      name: pageName,
-    }),
-  ).toBeVisible()
-})
-
-Then('I expect that the address {string} is visible', async ({ commonDataProvider, basePage }, dataKey) => {
-  const addressStr = shortenAddress(getDataByKey(commonDataProvider.commonData, dataKey))
-  await expect(basePage.getPage().getByText(addressStr)).toBeVisible()
-})
-
-Then(
-  'I expect that the address {string} with order {int} is invisible',
-  async ({ commonDataProvider, basePage }, dataKey, order) => {
-    const addressStr = shortenAddress(getDataByKey(commonDataProvider.commonData, dataKey))
-    await expect(basePage.getPage().getByText(addressStr).nth(order)).toBeVisible()
-  },
-)
+// Then('I expect that I go back to the {string} page', async ({ basePage }, pageName) => {
+//   // Check if whether we are on vaults page
+//   await expect(
+//     basePage.getPage().getByRole('heading', {
+//       name: pageName,
+//     }),
+//   ).toBeVisible()
+// })
+
+// Then('I expect that the address {string} is visible', async ({ commonDataProvider, basePage }, dataKey) => {
+//   const addressStr = shortenAddress(getDataByKey(commonDataProvider.commonData, dataKey))
+//   await expect(basePage.getPage().getByText(addressStr)).toBeVisible()
+// })
+
+// Then(
+//   'I expect that the address {string} with order {int} is invisible',
+//   async ({ commonDataProvider, basePage }, dataKey, order) => {
+//     const addressStr = shortenAddress(getDataByKey(commonDataProvider.commonData, dataKey))
+//     await expect(basePage.getPage().getByText(addressStr).nth(order)).toBeVisible()
+//   },
+// )
```

Comment:
nit: also this ?

Reply:
Sure, but I think from lines 312 to 323, it's not a common step that can be used for other projects, just only for Volta, right?

Reply:
Yes, i think so. If you can't execute the code with them, it mean it's not common. Let's remove them